src/applications/financial-status-report/components/resolution/ResolutionCommentsReview.jsx

```diff
@@ -11,11 +11,14 @@ const ResolutionCommentsReview = ({ data }) => {
           Personal statement
         </h4>
       </div>
-      <div className="review">
+      <dl className="review">
         <div className="review-row" key="resolution-comments">
-          {additionalData?.additionalComments}
+          <dt className="vads-u-display--none" />
```

Comment:
Would there be any accessibility value with using the `sr-only` class instead of `vads-u-display--none` and adding text to the `dt` element for a screenreader? ie. "Additional Comments"

Reply:
@jamigibbs, that's a great idea. I'll go ahead and do that.